src/libs/Localize/index.js

```diff
@@ -2,12 +2,27 @@ import _ from 'underscore';
 import lodashGet from 'lodash/get';
 import Str from 'expensify-common/lib/str';
 import * as RNLocalize from 'react-native-localize';
+import Onyx from 'react-native-onyx';
 import Log from '../Log';
 import Config from '../../CONFIG';
 import translations from '../../languages/translations';
 import CONST from '../../CONST';
 import LocaleListener from './LocaleListener';
 import BaseLocaleListener from './LocaleListener/BaseLocaleListener';
+import ONYXKEYS from '../../ONYXKEYS';
+
+// Current user mail is needed for handling missing translations
+let userEmail = '';
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    waitForCollectionCallback: true,
```

Comment:
@dukenv0307 `SESSION` onyx key is not a collection key, I'm wondering why `waitForCollectionCallback: true` was added?

Reply:
That was probably a bad copy pasta we have missed

Reply:
Np, we will remove it in a TS migration PR 👍